java/examples/pom.xml

```diff
@@ -50,14 +50,10 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <version>${hadoop.version}</version>
-      <scope>compile</scope>
```

Comment:
This causes another regression. https://github.com/apache/orc/pull/808 aims to fix this.